res/values-vi/strings.xml

```diff
@@ -1116,17 +1178,30 @@
     <string name="Invites">Những lời mời</string>
     <string name="Show_Spectator_Count">Hiện số lượng người xem</string>
     <string name="TRY_IT_">THỬ ĐI!</string>
+    <string name="skin_trial_info" translatable="true">You can try this skin for one game. After you die or change, it will go away. When you are trying a skin, no one else can see it.\nPrice: %s plasma.</string>
```

Comment:
This one is also not based on the English file; a comma has been included for polished reading.